source/_components/device_tracker.geofency.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Geofency"
+description: "Instructions for how to use Geofency to track devices in Home Assistant."
+date: 2017-08-22 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: geofency.png
+ha_category: Presence Detection
+ha_release: 0.52
+---
+
+This platform allows you to detect presence using [Geofency](http://www.geofency.com/). Geofency is an [paid app](https://itunes.apple.com/app/id615538630) for iOS that allows users to configure a request that will be sent when a geofence or iBeacon region is entered or exited. This can be configured with Home Assistant to update your location.
+
+To integrate Geofency in Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: geofency
+```
+
+Configuration variables:
+
+- **mobile_beacons** (*Optional*): List of beacon names that are to be treated as *mobile*. The name must match the name you configure in Geofency. By default beacons will be treated as *stationary*.
+
+A full sample configuration for the `geofency` platform is shown below:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: owntracks
+    mobile_beacons:
+      - car
+      - keys
+```
+
+To configure Geofency, you must configure (via the Webhook feature) to send a POST request to your Home Assistant server at `http://<ha_server>/api/geofency`. Use the default POST format, and make sure to include the API password if you have configured a password in Home Assistant (add `?api_password=<password>` to the end of the URL). Make sure to enable the 'Update Geo-Position' functionality for mobile beacons.
+
+The device tracker name used for geofences will be automatically generated by Geofency, and you will find it in `known_devices.yaml` after the first request. For beacons, the device name will be `beacon_<name from Geofency>`, e.g. `device_tracker.beacon_car`. 
```

Comment:
*Spelling & Grammar*:
- `The device tracker name used for geofences will be automatically generated by Geofency` -> `Geofency will automatically generate the device tracker name used for geofences` (suggestion)
- `e.g.` -> `e.g.,` (missing comma)